src/main/java/gift/exception/GlobalExceptionHandler.java

```diff
@@ -64,6 +64,19 @@ public ResponseEntity<ExceptionResponseDto> handleUserNotFoundException(
         return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(exception);
     }
 
+    @ResponseBody
+    @ExceptionHandler(UnauthorizedWishListException.class)
+    public ResponseEntity<ExceptionResponseDto> handleUnauthorizedWishList(
+        UnauthorizedWishListException e) {
+        List<String> errors = new ArrayList<>();
+        errors.add(e.getMessage());
+        ExceptionResponseDto exception = new ExceptionResponseDto(
+            errors,
+            LocalDateTime.now());
```

Comment:
아래와 같이 한건의 error message만 에러 응답으로 넣어주는 것이라면 `List.of` 와 같은 [컬렉션 팩토리](https://devbksheen.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%EC%BB%AC%EB%A0%89%EC%85%98-%ED%8C%A9%ED%86%A0%EB%A6%ACCollection-Factory-%EC%86%8C%EA%B0%9C-%EB%B0%8F-%EC%82%AC%EC%9A%A9%EB%B2%95)를 활용해볼 수 있을 것 같아요. 

```suggestion
    public ResponseEntity<ExceptionResponseDto> handleUnauthorizedWishList(UnauthorizedWishListException e) {
        ExceptionResponseDto exception = new ExceptionResponseDto(
            List.of(e.getMessage()),
            LocalDateTime.now()
        );
```

Reply:
생각해보면 지금의 케이스 외에도 하나의 errorMessage만 넣어주는 절차가 불필요하게 긴 것 같은데, `ExceptionResponseDto` 에 단건의 error message만 받는 [정적 팩토리 메서드](https://tecoble.techcourse.co.kr/post/2020-05-26-static-factory-method/)를 만들어서 대체해보면 어떨까요?